modules/manage/pages/cluster-maintenance/manage-throughput.adoc

```diff
@@ -6,11 +6,13 @@ Redpanda supports applying throughput throttling on both ingress and egress inde
 
 == Throughput throttling enforcement
 
+NOTE: As of v24.2, Redpanda enforces all throughput limits per broker, including client throughput.  
+
 Throughput limits are enforced by applying backpressure to clients. When a connection is in breach of the throughput limit, the throttler advises the client about the delay (throttle time) that would bring the rate back to the allowed level. Redpanda starts by adding a `throttle_time_ms` field to responses. If that isn't honored, delays are inserted on the connection's next read operation. The throttling delay may not exceed the limit set by xref:reference:tunable-properties.adoc#max_kafka_throttle_delay_ms[`max_kafka_throttle_delay_ms`].
 
 == Broker-wide throughput limits
 
-Broker-wide throughput limits account for all Kafka API traffic going in to or out of the broker. The limit values represent the allowed rate of data in bytes per second passing through in each direction. Redpanda also provides administrators the ability to exclude clients from throughput throttling and to fine-tune which Kafka request types are subject to throttling limits.
+Broker-wide throughput limits account for all Kafka API traffic going into or out of the broker. The limit values represent the allowed rate of data in bytes per second passing through in each direction. Redpanda also provides administrators the ability to exclude clients from throughput throttling and to fine-tune which Kafka request types are subject to throttling limits.
```

Comment:
This section is about the node-wide (not client id specific) ingress/egress quotas. They do support exclusions but the client quotas didn't and still don't.

Reply:
my bad, thanks. But are these Kafka API configs or just still cluster configs?  

Might get confusing if we dont specify.

Reply:
Still just cluster configs. The one controlling exclusions specifically is [kafka_throughput_control](https://docs.redpanda.com/current/reference/properties/cluster-properties/#kafka_throughput_control). This one, as well as the other ingress/egress cluster configs are listed in the section below the one this comment links to. Maybe the title of it would be better as "Broker-wide throughput limit ~~properties~~ *cluster configs*"